apps/prairielearn/elements/pl-matching/pl-matching.css

```diff
@@ -47,3 +47,12 @@
   background-color: rgba(0, 0, 0, 0.03);
   margin-right: 0.5rem;
 }
+
+.pl-matching-container-bottom {
+  display: block;
+}
+
+.pl-matching-container-bottom .pl-matching-statement-select,
+.pl-matching-container-right .pl-matching-statement-select {
+  min-width: 4.5em;
```

Comment:
There is a `min-width` already set in line 19. Can you clarify why this one is necessary?

Reply:
Yes, when I tried the existing code with longer statements and values, it partially hides the value selected in the drop-down select element. This behvaiour happens even with the previous examples. Should I proceed too change it directly in line 19, or keep this current config so it will be applies only to the long statement examples?

Reply:
I just tested locally and didn't have any issues. The default 4em is enough for the example course question, especially since all it needs to show is a letter or digit, unless you're using the `full-text` option (which is probably a bad idea if you have a long option text).